app/apptesting/test_helpers.go

```diff
@@ -3,12 +3,16 @@ package apptesting
 import (
```

Comment:
We should review these changes in more depth soon, but I think they're ok for now